js/level1.js

```diff
@@ -0,0 +1,30 @@
+//layout for blockage
+var map = [
+[5,1,9,1,1,6],
+[4,0,6,4,3,2],
+[10,4,0,2,5,2],
+[4,3,0,0,3,2],
+[4,6,4,2,5,0],
+[7,3,3,3,3,8]
+]
+
+//array indicating the exits
```

Comment:
I think it's not necessary to split these two arrays (`map` and `exit`), but good that you justify why in the presentation.